templates/pipeline/structured_detailed_error_counts_v1/view.sql

```diff
@@ -0,0 +1,36 @@
+WITH error_examples AS (
+  SELECT
+    TIMESTAMP_TRUNC(submission_timestamp, HOUR) AS hour,
+    document_namespace,
+    document_type,
+    document_version,
+    error_type,
+    error_message,
+    udf_js_gunzip(ARRAY_AGG(payload)[OFFSET(0)]) AS sample_payload,
```

Comment:
Open to alternatives that don't require aggregating all the payloads (where is `arbitrary` when you need it?)

Reply:
Does ANY_VALUE work here?

Reply:
Aha, that's just what I was looking for, thanks!